miden-tx/src/lib.rs

```diff
@@ -21,6 +21,8 @@ mod compiler;
 pub use compiler::{ScriptTarget, TransactionCompiler};
 
 mod executor;
+#[cfg(feature = "testing")]
+pub use executor::testing::MockDataStore;
```

Comment:
should we export a `testing` module like in other crates?

Reply:
This was replaced by the flatter structure